### PR TITLE
How-to guide on Uninstalling Radius 

### LINKED ISSUE #988: Document uninstallation for the `rad` CLI
## What content needs to be created or modified?

We should have documentation for removing the Radius CLI. This was mentioned via discord: https://discord.com/channels/1113519723347456110/1180199615782453318

[AB#10634](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/10634)

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.io/community/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Adds docs on removing Radius

### Summary
:memo::twisted_rightwards_arrows::wastebasket:

This pull request updates the documentation for Kubernetes operations, by adding a new page for uninstalling Radius, adding a prerequisites section for installing Radius, and changing the order of the metadata page in the sidebar.

> _`Kubernetes docs`_
> _Adding, updating, removing_
> _Autumn of changes_

### Walkthrough
*  Add a new section for prerequisites to the Kubernetes install page ([link](https://github.com/radius-project/docs/pull/996/files?diff=unified&w=0#diff-5b1e8b14bb206354b9eccd6444ecf75fc7c08aa75b63c229da510b4ea8ff1345R14-R19))
*  Change the weight of the Kubernetes metadata page to appear after the Kubernetes install page in the sidebar navigation ([link](https://github.com/radius-project/docs/pull/996/files?diff=unified&w=0#diff-3c30f4ddc51c75b9980a6ad8453e01453fdd4cadf996b381620b65ffd855a736L6-R6))
*  Create a new page for Kubernetes uninstall with front matter and three sections ([link](https://github.com/radius-project/docs/pull/996/files?diff=unified&w=0#diff-dcce6478af092e258aa498c466e5e30ed8843b049fa7a98f00f5f6e321eeef4dL1-R33))

## Issue reference
Fixes: #988

### CODE REVIEW

File: docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+All the Radius services running in the `radius-system` namespace will be deleted.
+
+## Step 2: Delete the `radius-system` namespace
+
+To delete the `radius-system` namespace, run the following command:
+
+```bash
+kubectl delete namespace radius-system
+```
+
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md:32]:
```suggestion

All Radius configuration and data will be removed as part of the namespace. This completely removes from your cluster.

```

File: docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- [Radius installed on Kubernetes cluster]({{< ref "guides/operations/kubernetes/kubernetes-install" >}})
+
+## Step 1: Uninstall the Radius control-plane from your kubernetes cluster
+
+To uninstall the existing Radius control-plane, run the following command:
+
+```bash
+rad uninstall kubernetes
+```
+All the Radius services running in the `radius-system` namespace will be deleted.
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md]:
```suggestion

All the Radius services running in the `radius-system` namespace will be removed. Note that Radius configuration and data will still be persisted in the cluster until the namespace is also deleted.
```

File: docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+type: docs
+title: "How-To: Uninstall Radius"
+linkTitle: "Uninstall Radius"
+description: "Learn how to uninstall Radius control plane and rad cli"
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md]:
```suggestion
description: "Learn how to uninstall Radius control plane from your Kubernetes cluster"
```

File: docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+type: docs
+title: "How-To: Uninstall Radius"
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md]:
```suggestion
title: "How-To: Uninstall Radius from Kubernetes"
```

File: docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+To delete the `radius-system` namespace, run the following command:
+
+```bash
+kubectl delete namespace radius-system
+```
+
+## Step 3: Remove the rad CLI
+
+You can remove the rad CLI by deleting the binary and ~/.rad folder from your machine.
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md]: I think a dedicated how-to guide under the tooling section would be helpful. We can link to that via a "Further reading" section here

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Looking good! Some quick requests

File: docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md (hunk 5 above)
Comment (Reshrahim) [reply docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md]: Not sure if this requires a dedicated a "how-to" for removing rad cli as it is simply deletion of the binary and rad folder

File: docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+```bash
+kubectl delete namespace radius-system
+```
+
+All Radius configuration and data will be removed as part of the namespace. This completely removes from your cluster.
+
+## Step 3: Remove the rad CLI
+
+You can remove the rad CLI by deleting the binary and ~/.rad folder from your machine.
```
Comment (AaronCrawfis) [inline docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md]: Could you add bullets for Linux/Windows, or link to the CLI documentation that shows where the path is: `/guides/tooling/rad-cli/overview/#binary-location`

Comment (Reshrahim) [inline docs/content/guides/operations/kubernetes/kubernetes-uninstall/index.md]:
```suggestion
You can remove the rad CLI by deleting the [binary]
({{< ref "/guides/tooling/rad-cli/overview/#binary-location" >}}) and ~/.rad folder from your machine.
```

File: docs/content/getting-started/index.md [hunk 7]
```diff
@@ -98,7 +98,7 @@ Congrats! You're running your first Radius app. When you're ready to move on to
 
 ## 5. Add Database
 
-This step will add a database (Redis Cache) to the application.
+his step will add a database (Redis Cache) to the application.
```
Comment (Reshrahim) [inline docs/content/getting-started/index.md]:
```suggestion
This step will add a database (Redis Cache) to the application.
```

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**